analytics_data_api/v0/views/learners.py

```diff
@@ -28,7 +28,7 @@
 
 class LearnerView(CourseViewMixin, generics.RetrieveAPIView):
     """
-    Get a particular student's data for a particular course.
+    Get data for a particular learner in a particular course.
```

Comment:
Doc team rule o' thumb: users = everyone with an account, learners = enrolled non-privileged users. When in doubt, users carries the day. I'm happy to change everything to user, but this is pretty much the result of a search and replace of student since we no longer prefer that term...